src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -325,7 +326,7 @@ private void cleanUpZipFile() {
         zip_properties.put("create", "false");
 
         // Specify the path to the ZIP File that you want to read as a File System
```

Comment:
Could you update this comment with some language about the change allowing cross-platform use, e.g., `(File#toURI allows this to work for different operating systems, including Windows)`?

Reply:
Sure. I've added the comment as requested.